integreat_cms/cms/templates/push_notifications/push_notification_list_row.html

```diff
@@ -34,7 +34,11 @@
         </div>
     </td>
     <td class="pr-2">
-        {{ push_notification_translation.last_updated }}
+        {% if push_notification_translation.last_updated %}
+            {{ push_notification_translation.last_updated }}
+        {% else %}
+            <i icon-name="minus"></i>
+        {% endif %}
```

Comment:
This is "last updated" column, not "Planned send date"? 🤔

Reply:
Oh no! You're right!

Reply:
I'm sorry! Now it should be correct!